*Name:* Thresa Kelly

*Professor:* Dr. Allison Kirkpatrick

*Date:* 12/2/21

*Science Goal:* Which galaxies in the EGS field are likely AGN, and what are their physical properties?

*Details:* CEERS is an approved project on JWST to observe the Extended Groth Strip with the NIRCam, NIRSpec, and MIRI instruments. Thresa is going to identify AGN in the MIRI field of view. She will plot their spectral energy distributions, look for spectra in the 3D-HST database, look for radio observations through the FIRST database, and make images.



* Milestone: Read the .csv file

In [6]:
# imports
from astropy.io import ascii

# read data from file 
data = ascii.read("EGS_for_Thresa.csv")

# view data
print(data.columns, '\n')
print(data)

<TableColumns names=('ID','RA','Dec','Redshift','u','u Unc','g','g Unc','r','r Unc','i','i Unc','z','z Unc','y','y Unc','IRAC Ch 1','Ch 1 Unc','IRAC Ch 2','Ch 2 Unc','IRAC Ch 3','Ch 3 Unc','IRAC Ch 4','Ch 4 Unc','PACS 100','100 Unc','PACS 160','160 Unc','SPIRE 250','250 Unc','SPIRE 350','350 Unc','SPIRE 500','500 Unc')> 

             ID                 RA        Dec    ... SPIRE 500  500 Unc 
--------------------------- ---------- --------- ... --------- ---------
HELP_J141411.189+520025.636 213.546622 52.007121 ...       NaN       NaN
HELP_J141411.417+515923.419 213.547572 51.989839 ...       NaN       NaN
HELP_J141411.539+515959.135 213.548077  51.99976 ...       NaN       NaN
HELP_J141411.902+515927.909 213.549592 51.991086 ...       NaN       NaN
HELP_J141412.159+515941.025 213.550661 51.994729 ...       NaN       NaN
HELP_J141412.263+515904.194 213.551097 51.984498 ...       NaN       NaN
HELP_J141412.597+520041.294 213.552488 52.011471 ...  7254.822   5038.43
HELP_J141412.781+51

* Milestone: Which wavelengths correspond with which filters in the .csv file?



'u' = ultraviolet: λeff= 365 nm, Δλ= 66 nm

'g' = green: λeff= 464 nm, Δλ= 128 nm

'r' = red: λeff= 658 nm, Δλ= 138 nm

'i' = infared: λeff= 806 nm, Δλ= 149 nm

'z' = infared: λeff= 900 nm

'y' = infared: λeff= 1020 nm, Δλ= 120 nm

'IRAC Ch 1' =  Infrared Array Camera channel 1: λcenter= 3.56 um

'IRAC Ch 2' =  Infrared Array Camera channel 2: λcenter= 4.54 um

'IRAC Ch 3' =  Infrared Array Camera channel 3: λcenter= 5.73 um

'IRAC Ch 4' =  Infrared Array Camera channel 4: λcenter= 7.91 um

'PACS 100' = Photodetector Array Camera and Spectrometer 

'PACS 160' = Photodetector Array Camera and Spectrometer

'SPIRE 250' = Photometer: 250 um

'SPIRE 350' = Photometer: 350 um

'SPIRE 500' = Photometer: 500 um

* Filters: https://en.wikipedia.org/wiki/Photometric_system
* IRAC paper: https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.145.2828&rep=rep1&type=pdf#:~:text=IRAC%20is%20a%20four%2Dchannel,%3B%204.5%20and%208%20m).
* PACS: https://www.cosmos.esa.int/web/herschel/pacs-overview 
* Spire: https://www.cosmos.esa.int/web/herschel/first-light

* Milestone: Select galaxies in the MIRI field of view